src/System.Text.Json/tests/Serialization/SampleTestData.BlogPost.cs

```diff
@@ -0,0 +1,367 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Net;
+
+namespace System.Text.Json.Serialization.Tests.Schemas.BlogPost
+{ 
+    // This schema populated using blogpost structure of https://medium.com
```

Comment:
Are there any copyright / proprietary / licensing concerns about doing so?

To be safe, I'd suggest instead basing it on something from bing or Azure or the dev blogs or some such thing, or even better just making something up.

Reply:
> Are there any copyright / proprietary / licensing concerns about doing so?

Not sure, not likely as any app would use the json structure to talk to this site
> To be safe, I'd suggest instead basing it on something from bing or Azure or the dev blogs or some such thing, or even better just making something up.

Sure I will look up for a big json schema from them and refactor the code

Reply:
I already gave this feedback a couple months ago. Don't use content you find on the web in tests unless you know it is licensed for general use. Using blog content is even more challenging because there can be other posts on the same blog that are offensive (some of which might not be posted yet). It is very challenging to use content from medium and similar services. When in doubt, ask me or someone else who understands these issues first.

Feedback I gave earlier: https://github.com/dotnet/corefx/pull/39668#discussion_r314833357

Reply:
To clarify, the tests don't use any "content" of any of the blogs published on medium. Only the schema/class definition is related to medium (i.e. the schema that would be required to load a JSON response from calling the medium REST API endpoint). Once we have the class, it's properties are manually populated with dummy data/content.

> Sure. It is fine to use an API for a commercial service, like medium or Netflix.

From your earlier comment, this PR is doing exactly that. The concerns that were brought up earlier were all related to the content itself (which have been addressed), and not the schema.

That said, I'd be OK with us not even using the medium developer APIs to generate classes (and use bing/azure/etc.), but I don't see it being absolutely necessary. I think there is value in testing popular/commonly used "real-world" end points and making sure the `JsonSerializer` can handle such classes/models. How can we avoid any references to those end-points in the tests if we want coverage for real scenarios. Is the suggestion to forgo that goal, and if not, what's the recommendation?

Reply:
Right, I have addressed @richlander's comment and updated the content, this is about the json schema/structure

> To be safe, I'd suggest instead basing it on something from bing or Azure or the dev blogs or some such thing, or even better just making something up.

By my understanding this is suggesting `just in case` use internal web/blog payloads or make up them. I am not planning to make up the schema as all other test schemas used in json tests are made up and with this one wanted to have one real world like scenario just like @ahsonkhan mentioned. So thinking to lookup payloads from internal sites and update as needed

Reply:
We cannot use the schema of a commercial service unless there is a license that allows us to do so.

Reply:
I don't know, there is only info about contents nothing about public API payloads and don't think would find anything. Anyways as I said I will find payloads from `bing or Azure or the dev blogs` and use, as they are Microsoft product assume have no issue with licensing right?

Reply:
Not exactly. This is an open source project, so using proprietary MS content is also a problem. Only use content that has a favorable license or terms of service and share a link with me to such a document.

Reply:
Sure will do, thanks

Reply:
`bing or Azure or the dev blogs` doesn't seem to be open source. Even there is many knows open source blog/ecommerce platforms/CMSs, the web app built on them are not really open source, plus there is no any licensing/copyright info about how their Rest API payloads could be used. I was thinking to use Wikipedia as it is open source but didn't found info about using Rest API payload schema usage. I have talked about it with @richlander few days ago and API payload schema/structure licensing is such grey area we better not use in our code. So I am planning to just make up a schema of big object as @stephentoub suggested